src/test/java/fi/otavanopisto/kuntaapi/test/server/unit/images/ImageScalerTest.java

```diff
@@ -19,29 +19,121 @@
 @RunWith (CdiTestRunner.class)
 public class ImageScalerTest {
   
-  
   @Inject
   private ImageReader imageReader;
-
+  
   @Inject
   private ImageScaler imageScaler;
 
   @Test
-  public void testScaleMaxSize() throws IOException {
+  public void testScaleToFitLandscape() throws IOException {
     assertNotNull(imageScaler);
-    try (InputStream imageStream = getClass().getClassLoader().getResourceAsStream("test-image-1000.jpg")) {
+    try (InputStream imageStream = getClass().getClassLoader().getResourceAsStream("test-image-1000-667.jpg")) {
       BufferedImage bufferedImage = imageReader.readBufferedImage(imageStream);
       assertNotNull(bufferedImage);
       
       assertEquals(1000, bufferedImage.getWidth());
       assertEquals(667, bufferedImage.getHeight());
       
-      BufferedImage scaledImage = imageScaler.scaleMaxSize(bufferedImage, 400);
+      BufferedImage scaledImage = imageScaler.scaleToFit(bufferedImage, 400);
       
       assertEquals(400, scaledImage.getWidth());
       assertEquals(266, scaledImage.getHeight());
     }
+  }
+
+  @Test
+  public void testScaleToFitPortrait() throws IOException {
+    assertNotNull(imageScaler);
+    try (InputStream imageStream = getClass().getClassLoader().getResourceAsStream("test-image-667-1000.jpg")) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "test-image-667-1000.jpg" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)